lib/src/main/java/net/mossol/bot/storage/MenuStorageService.java

```diff
@@ -0,0 +1,24 @@
+package net.mossol.bot.storage;
+
+import java.util.List;
+import java.util.Map;
+
+import net.mossol.bot.model.LocationInfo;
+import net.mossol.bot.service.MenuServiceHandler.FoodType;
+
+public interface MenuStorageService {
```

Comment:
`MenuStorageService` naming is a little confusing... @.@ I understand you named it to say that the `MenuStorageService` serves as 'storage', not having any additional business logic. 

But `MenuService` is also enough! I think. HDYT?

Reply:
But we already have `MenuServiceHandler`. If we rename to `MenuService`, I think it causes more confuse. 
And, there was some extra business logic coming from separating logic between CentralDogma and MongoDB. I updated PR removing all logics related with CentralDogma. How about now?

Reply:
look good~ it became more clear!